### PR TITLE
IOS-313 measurements service

### PR DESCRIPTION
**Pull request by viikufa**
refactored code with using measurement service

### CODE REVIEW

File: station/Classes/Model/SQLite/RuuviTagDataSQLite.swift [hunk 1]
```diff
@@ -50,8 +50,10 @@ extension RuuviTagDataSQLite: FetchableRecord {
         if let celsius = Double.fromDatabaseValue(row[RuuviTagDataSQLite.celsiusColumn]) {
             temperature = Temperature(value: celsius, unit: .celsius)
             if let relativeHumidity
-                = Double.fromDatabaseValue(row[RuuviTagDataSQLite.relativeHumidityInPercentColumn]) {
-                humidity = Humidity(c: celsius, rh: relativeHumidity)
+                = Double.fromDatabaseValue(row[RuuviTagDataSQLite.relativeHumidityInPercentColumn]),
+                let temperature = temperature {
+                humidity = Humidity(value: relativeHumidity,
```
Comment (rinat-enikeev) [inline station/Classes/Model/SQLite/RuuviTagDataSQLite.swift:55]: either contract is relativeHumidityInPercent and Humidity consumes percents, or the name of the column must be relativeHumidityInFractionOfOne

File: station/Classes/Model/SQLite/RuuviTagDataSQLite.swift [hunk 2]
```diff
@@ -85,7 +87,7 @@ extension RuuviTagDataSQLite: PersistableRecord {
         container[RuuviTagDataSQLite.macColumn] = macId?.value
         container[RuuviTagDataSQLite.rssiColumn] = rssi
         container[RuuviTagDataSQLite.celsiusColumn] = temperature?.converted(to: .celsius).value
-        container[RuuviTagDataSQLite.relativeHumidityInPercentColumn] = humidity?.rh
+        container[RuuviTagDataSQLite.relativeHumidityInPercentColumn] = humidity?.value
```
Comment (rinat-enikeev) [inline station/Classes/Model/SQLite/RuuviTagDataSQLite.swift:90]: same here, confusing database column. 

Please also check migration.

File: station/Enums/Language.swift [hunk 3]
```diff
@@ -20,4 +21,26 @@ extension Language {
...
+        case .swedish:
+            return Locale(identifier: "sv")
+        }
+    }
+
+    var humidityLanguage: HumiditySettings.Language {
+        switch self {
+        case .russian:
+            return .ru
+        default:
```
Comment (rinat-enikeev) [inline station/Enums/Language.swift:42]: shy not handling other cases? Do you need help with translation or it is intentional?

**Review by rinat-enikeev — CHANGES_REQUESTED** [review]

File: station/Enums/Language.swift (hunk 3 above)
Comment (viikufa) [reply station/Enums/Language.swift:42]: it is international

**Pull request merged**